nixos/modules/services/mail/dovecot.nix

```
@@ -93,7 +93,7 @@ let
   # The idea is to match everything that looks like `$term =`
   # but not `# $term something something`
   # or `# $term = some value` because those are comments.
-  configContainsSetting = lines: term: (match "^[^#]*\b${term}\b.*=" lines) != null;
+  configContainsSetting = lines: term: (match "^[^#]*${term}[[:blank:]]*=.*" lines) != null;
```

@piegamesde:
> @2xsaiko Nix does not know about `\b` (or any other escape sequence), what's the intent of the regex here?

I didn't write that, but I think this now almost does what is intended. It matches "xfoo =" for term="foo" though. I would write

```suggestion
  configContainsSetting = lines: term: (match "[[:blank:]]*${term}[[:blank:]]*=.*" lines) != null;
```

@RaitoBezarius what do you think? Looks like you wrote this originally.

Good to know it doesn't support these escapes.

---

So my version more closely matches the original regex which has the same over-matching, but I'm unsure if it is intended like that or not. I read up on the dovecot configuration docs, still unclear.